qemu/tests/cfg/multi_host.cfg

```diff
@@ -15,7 +15,7 @@
             vms = "vm1"
             start_vm = no
             migration_test_command = help
-            migration_bg_command = "cd /tmp; nohup tcpdump -q -i any -t ip host localhost"
+            migration_bg_command = "cd /tmp; tcpdump -q -i any -t ip host localhost &"
             migration_bg_check_command = pgrep tcpdump
             migration_bg_kill_command = pkill -9 tcpdump
             kill_vm_on_error = yes
```

Comment:
variants mig_protocol:
                #Migration protocol.
                - tcp:
                - x-rdma:
                - rdma:
                - fd:
                - exec:


====> could you add protocol to every variant